ci/recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - q2-types {{ release }}.*
     - scikit-bio
     - scipy
-    - unifrac
+    - unifrac >=0.20.0
```

Comment:
@wasade - what do you think of setting a minimum version for unifrac? If "yes" - is this the right version?

Reply:
:+1: